About/About.xml

```diff
@@ -5,6 +5,7 @@
   <author>PorqueNoLosDos</author>
   <packageId>PorqueNoLosDos.ColorCodedMoodBar</packageId>
   <supportedVersions>
+    <li>1.0</li>
```

Comment:
I remember causing issues with 1.1 if users activated my mod without also having Harmony installed. It'd crash the game and clear their mod list, so I think I'd rather not support it directly unless that's fixed.
```suggestion
```

Reply:
Fine by me. Doubtful many people even use 1.0 any more